test/parallel/test-require-cache.js

```diff
@@ -21,3 +21,14 @@ var assert = require('assert');
 
   assert.strictEqual(require(relativePath), fakeModule);
 }
+
+
+{
+  const relativePath = '../fixtures/semicolon';
+  const mod = require(relativePath);
+  mod.extraProperty = {};
+
+  assert.strictEqual(require(relativePath).extraProperty, mod.extraProperty);
+  require.cache = {};
+  assert.equal(typeof require(relativePath).extraProperty, 'undefined');
```

Comment:
Any reason for not using `assert.strictEqual(require(relativePath).extraProperty, undefined);`?